dist/runCall.js

```diff
@@ -229,8 +396,8 @@ module.exports = function (opts, cb) {
 
   function createEthereumConfig() {
     var ethereumConfig = new msg_pb.EthereumConfig();
-    ethereumConfig.setMaxcodesize(new Uint8Array([96, 0]));
-    ethereumConfig.setAccountstartnonce(new Uint8Array([0]));
+    ethereumConfig.setMaxcodesize(new Uint8Array([127, 255, 255, 255]));
```

Comment:
Where is this number `[127, 255, 255, 255]` coming from? I see the same number used for functions like `setPetersburgblocknumber` and setConstantinopleblocknumber`. It looks like it's being used as a `true` flag or something?

Reply:
This line and the next one were modified for debugging purposes and I forgot to revert the changes. The number `[127, 255, 255, 255]` was chosen to avoid a block number collision (see [PR#3](https://github.com/runtimeverification/ethereumjs-vm/pull/3#discussion_r288157720)).